app/src/main/res/layout/previous_button_fragment.xml

```diff
@@ -10,14 +10,14 @@
         android:id="@+id/textview_second"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        app:layout_constraintBottom_toTopOf="@id/button_second"
+        app:layout_constraintBottom_toTopOf="@id/edtPrevious"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
     <Button
-        android:id="@+id/button_second"
-        android:layout_width="wrap_content"
+        android:id="@+id/edtPrevious"
+        android:layout_width="0dp"
```

Comment:
if the button is the only thing on a screen, make it large.

Reply:
Be careful about using absolute measurements (0dp) versus relative (wrap_content).  Relative is better for flexible resolutions.